src/Entity/Review.php

```diff
@@ -116,9 +116,14 @@ public function getCreatedAt(): ?\DateTimeImmutable
         return $this->createdAt;
     }
 
-    public function setCreatedAt(\DateTimeImmutable $createdAt): static
+    public function setCreatedAt(\DateTimeImmutable|\DateTime $createdAt): static
     {
-        $this->createdAt = $createdAt;
+        if ($createdAt instanceof \DateTime) {
```

Comment:
I added it because if type `DateTime` was passed to the `createdAt` property, it complained since it's not of type `DateTimeImmutable`. Because of this I wasn't able to use the `<dateTimeBetween()>` faker function in `fixtures/review.yaml`

Reply:
Just to test, could you revert this change and try:
'<(\DateTimeImmutable::createFromMutable(dateTimeBetween("-20 years", "now")))>'

Reply:
I already tried using the `createFromMutable()` function, but for some reason it still complained that the type received was `DateTime`. Reverting the changes and copy-pasting the line you suggested triggered the same error:
<img width="1117" alt="Screenshot 2023-11-08 at 16 00 28" src="https://github.com/damyancodixis/movie_rating/assets/133638859/bf57b727-840f-4693-a09b-15a1683398d4">